azure/requests.yaml

```diff
@@ -1,4 +1,12 @@
 releases:
+    - name: "> 13.0.0-beta1"
```

Comment:
I saw there is a beta version on-going, should I change it to `13.0.0` as normal?

Reply:
AIUI the beta release is a preview release for node pools on Azure.

I'm not sure. I think it's best to ask Celestial.